vendor/lowrisc_ip/ip/prim/prim_ram_1p_scr.core

```diff
@@ -31,4 +29,4 @@ targets:
   default:
     filesets:
       - tool_verilator ? (files_verilator_waiver)
-      - files_rtl
+      - files_rtl
```

Comment:
Same above for this vendored file (needs to be fixed upstream or with a vendoring patch) 
Also missing new line at end of file